submissions/karelskiy/HTML and CSS practice/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
```

Comment:
Check your markup in [W3CValidator](https://validator.w3.org/).
Fix all errors if you have.